From: Phil Elwell <phil@raspberrypi.com>
Date: Fri, 1 May 2020 14:58:23 +0100
Subject: SQUASH: spi: Demote SPI_CS_HIGH warning to KERN_DEBUG

This warning is unavoidable from a client's perspective and
doesn't indicate anything wrong (just surprising).

SQUASH with "spi: use_gpio_descriptor fixup moved to spi_setup"

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/spi/spi.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/spi/spi.c b/drivers/spi/spi.c
index 4a709ecee839..13cda766488a 100644
--- a/drivers/spi/spi.c
+++ b/drivers/spi/spi.c
@@ -3233,8 +3233,8 @@ int spi_setup(struct spi_device *spi)
 
 	if (ctlr->use_gpio_descriptors && ctlr->cs_gpiods &&
 	    ctlr->cs_gpiods[spi->chip_select] && !(spi->mode & SPI_CS_HIGH)) {
-		dev_warn(&spi->dev,
-			 "setup: forcing CS_HIGH (use_gpio_descriptors)\n");
+		dev_dbg(&spi->dev,
+			"setup: forcing CS_HIGH (use_gpio_descriptors)\n");
 		spi->mode |= SPI_CS_HIGH;
 	}
 
